src/lib/factories.tsx

```diff
@@ -105,18 +105,16 @@ function createShorthandFromValue(
   const valIsNoop = _.isNil(value) || typeof value === 'boolean'
   if (valIsNoop && !options.render) return null
 
+  // return value as is if it a ReactElement
```

Comment:
nit: this line is hard to be read from the first time :)

Would suggest small change:

> return value 'as is' if it is a ReactElement